Bài báo cáo của Phạm Thành Đạt

datkk06 opened the pull request:
- update 09/08/2016:
  - Lab NAT + Aliass trong 1 mô hình.
  - Lab pfSense services DHCP server.

datkk06 committed:
update  09/08/2016

ducnc commented:
Em cập nhật lại cấu trúc của repo như sau: 
- Phần readme phải thấy ngay được tất cả những thứ em đang làm (Mục lục)
- Bỏ cái ảnh trong readme, chuyển nó về dạng Markdown
- Trong trường hợp lab nhiều bài, cần tạo một thư mục `Lab` trong Repo. Mỗi bài lab là một file trong repo ấy
- Với mỗi file `md` phải được dẫn linh ở đâu đó trong `readme`.
- Nói chung phải thấy được sự liền mạch, hiện tại repo của em rời rạc

datkk06 committed:
update 12/8/2016

datkk06 committed:
update 12/08/2016

datkk06 committed:
update 12/08/2016

datkk06 committed:
update 12/08/2016

- Cập nhật lại các mục.
- Update NAT 1:1 pfSense.

datkk06 committed:
update 14/08/2016

- update lab NAT port forward

ducnc commented on DatPT/Pfsense/LAB/lab-NAT-portforward.md:
```diff
@@ -0,0 +1,44 @@
+
+##II. Thực hiện.
+
+- Bước đầu tiên là chúng ta sẽ tạo một vitual IP, đây là địa chỉ mà chúng ta dùng để public website ra ngoài.
```
Các bước lớn cần trình bày rõ ràng, làm nổi bật để tiện theo dõi
Ví dụ:  

#### Bước 1:.....

#### Bước 2:.....

ducnc commented on DatPT/Pfsense/LAB/lab-NAT-portforward.md:
```diff
@@ -0,0 +1,44 @@
+
+##I. Mô hình.
+
+![scr1](http://i.imgur.com/MkPSbr4.png)
```
Em vẽ lại mô hình, mô hình của em nhìn tủn mủn quá,
Lưu ý cập nhật IP vào các máy trong mô hình

ducnc commented on DatPT/Pfsense/LAB/lab-NAT-portforward.md:
```diff
@@ -0,0 +1,44 @@
+
+##III. Lưu ý:
+
+- Khi thực hiện lab chúng ta cần chú ý loại bỏ 2 check box trong phần `interfaces` => `WAN` như dưới hình :
```
Giải thích rõ tại sao lại làm như vậy

ducnc commented:
Em chú ý hạn chế tối đa các lỗi Typo. Ví dụ `overview` chứ không phải `overveiw`
Các link trong `readme` cần chú ý dẫn link chính xác 

**Lần này anh chưa thể merge được**

datkk06 commented:
a chưa merge nên e lấy link đồng bộ nên git của e sửa lại cho nên chắc có
thể sai a ạ.

Vào 15:51 Ngày 15 tháng 08 năm 2016, Nguyen Cong Duc <
notifications@github.com> đã viết:

> Em chú ý hạn chế tối đa các lỗi Typo. Ví dụ overview chứ không phải
> overveiw
> Các link trong readme cần chú ý dẫn link chính xác
> 
> _Lần này anh chưa thể merge được_
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/hocchudong/networking-team/pull/8#issuecomment-239756562,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/ANoiX-L7dHnBfOR2G9Q9u75dDqRle1Ufks5qgCihgaJpZM4JgMCn
> .

ducnc commented on DatPT/Pfsense/README.md:
```diff
@@ -1,3 +1,29 @@
+- Các chức năng: 
+ <ul>
+  <li>Aliass</li>
+  <li>[NAT 1:1](https://github.com/datkk06/networking-team/blob/master/hocchudong/Pfsense/LAB/lab-NAT-pfSense.md)</li>
```
Link chỗ này vẫn đang trỏ về `github.com/datkk06` nhé
Soi lại cho anh

datkk06 committed:
update 16/08/2016

- tổng quan về proxy server
- chức năng của proxy server
- lab chặn URL với proxy server trên pfSense

datkk06 committed:
update 16/08/2016

- Chỉnh sửa lại những gì còn lỗi như anh đã góp ý.
- Tổng quan về Proxy server
- Các chức năng của Proxy server
- Lab chặn URL bằng Proxy server trên pfSense

datkk06 committed:
Update 16/08/2016

- Chỉnh sửa lại những gì còn lỗi như anh đã góp ý.
+ Vẽ lại mô hình.
+ Thêm chú thích vào các bước.
- Tổng quan về Proxy server
- Các chức năng của Proxy server
- Lab chặn URL bằng Proxy server trên pfSense

`Về phần giải thích ở NAT port forward em không chắc lắm, có gì anh
update cho em về phần này với ạ`

datkk06 committed:
update 17/08/2016

Thực hiện thêm 3 bài lab trong phần proxy server:

- Loại bỏ một số host đặc biệt không bị chặn bởi Proxy.
- Chỉ được phép vào những trang web mà người quản trị cho phép.
- Thiết lập thời gian hiệu lực của rule.

datkk06 committed:
update 17/08/2016

Sửa lỗi Markdown
